tests/services/test_load_db.py

```diff
@@ -0,0 +1,180 @@
+import pytest
+from pymodm import connect
+
+from app.db.mongo_model_definitions import Service, User
+from app.services.load_db import LoadDb
+
+from dotenv import load_dotenv
+import os
+
+load_dotenv()
+
+
+class TestLoadDb:
+    @classmethod
+    def setup_class(cls):
+        connect(os.getenv("MONGO_DB_TEST_ENDPOINT"))
+
+    @classmethod
+    def teardown_class(cls):
+        Service.objects.all().delete()
+        User.objects.all().delete()
```

Comment:
This is a `unittest` way of doing things. In `pytest` use fixtures for database connection and database handle